api_tests/base/test_sloan_study.py

```diff
@@ -269,3 +269,11 @@ def test_user_override_cookie_false(self, app, user, preprint):
         cookies = resp.headers.getall('Set-Cookie')
 
         assert f' dwf_{SLOAN_COI_DISPLAY}=False; Domain=.osf.io; Path=/; samesite=None; Secure' in cookies
+
+    def test_browseable_api(self, app):
+        resp = app.get('/v2/?format=api')
+        assert resp.status_code == 200
+
+        headers = {'Content-Type': 'application/vnd.api+json'}
+        resp = app.get('/v2/?format=api', headers=headers)
```

Comment:
No, for this you want to do the same type of request that the browser is making in order to replicate the 502 that we're getting. Neither of these tests should have failed before your changes in this PR. But if you do one with an Accept header of `text/html` that will activate the browsable API in the way that the web browser does.

Reply:
You'll want to use the Accept header, not the Allow header.

Reply:
Ug I did actually try that `{'Accept': 'html/text'}`, but that 406ed, I think due to capitalization, but I found a correct president which is `{'accept': 'application/vnd.api+json;version=latest'}` Sorry this taking a much longer time then anticipated, but I'm having some docker or pytest problem and my local tests keep hanging indefinitely so I have no idea when/if they will complete.